cns/logger/log.go

```diff
@@ -61,8 +61,8 @@ func Close() {
 	}
 }
 
-func SetLogDirectory(dir string) {
-	Log.logger.SetLogDirectory(dir)
+func SetTargetLogDirectory(target int, dir string) error {
```

Comment:
I am assuming, the components in aquarius repo, that has vendor depedencies, will update it, when pull in the latest change or it was build break there?

Reply:
yes once vendor dep are updated